templates/default/babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["@babel/preset-env","next/babel"]
+    "presets": [
+    [
+      "next/babel", 
+      {
+        "preset-env": { "targets": { "node": "current" } }
```

Comment:
required to extend a class (aka the new Document() class)

Reply:
Is this the same as what we have already in the internal project?

Reply:
Yes it is, but it is done differently. we set `useBuiltIns` and we specify a `browserList` in the package.json which stated `node 10` which is the equivalent here at `node: current`.